topostats/io.py

```diff
@@ -6,15 +6,20 @@
 import numpy as np
 
 from pySPM.Bruker import Bruker
-from igor import binarywave  # this is already imported with afmformats but is being specialised for our use
-from afmformats import load_data
-from afmformats.mod_creep_compliance import AFMCreepCompliance
+from igor import binarywave
 from ruamel.yaml import YAML, YAMLError
 from ruamel.yaml.main import round_trip_load as yaml_load, round_trip_dump as yaml_dump
 from topostats.logs.logs import LOGGER_NAME
 
 LOGGER = logging.getLogger(LOGGER_NAME)
 
+try:
+    import libasd
+except ModuleNotFoundError:
+    LOGGER.warning(
+        "libasd module not installed. Ignore if '.asd' files are not being processed. \n Otherwise install via 'pip install .[libasd]' or if on an M1/2 Mac, follow libasd installation guidelines for 'build yourself' here: https://github.com/ToruNiina/libasd"
```

Comment:
Need to change the `pip install .[libasd]` since that is for installing from a locally cloned repository. Can also link directly to the build instructions of libasd

```suggestion
        "libasd module not installed. Ignore if '.asd' files are not being processed. \n Otherwise install via 'pip install topostats[libasd]' or if on an M1/2 Mac, follow libasd installation guidelines for 'build yourself' here: https://github.com/ToruNiina/libasd#building-by-yourself"
```

Reply:
removed due to libasd import now available